### PR TITLE
Gemini postgres

### PR DESCRIPTION
**Pull request by knewmanTE**
This separates Gemini into non-db and db tests and adds a Postgres test.

### CODE REVIEW

File: frameworks/Java/gemini/source_code [hunk 1]
```diff
@@ -27,9 +27,9 @@
 ./gemini/Docroot/WEB-INF/lib/mustache-compiler-0.8.13.jar
```
Comment (msmith-techempower) [inline frameworks/Java/gemini/source_code]: @knewmanTE We need to add the postgresql jar here as well.

**Comment by msmith-techempower** [comment]
@knewmanTE I'm not sure why the postgresql tests are failing, can you check this out on a completely blank repository clone to see what's missing?

**Comment by knewmanTE** [comment]
Yeah, I'll give that  shot.

**Comment by msmith-techempower** [comment]
Closed in favor of #1344

**Pull request closed without merging**